### ISSUE TITLE
Is not installed

### ISSUE THREAD

**Issue by DimitriBolt** [ropensci/git2r#444]
The error is:
```
In file included from libgit2/src/posix.h:155,
                 from libgit2/src/common.h:114,
                 from libgit2/src/runtime.h:10,
                 from libgit2/src/alloc.c:9:
libgit2/src/unix/posix.h:35:3: error: #error GIT_USE_NSEC defined but unknown struct stat nanosecond type
   35 | # error GIT_USE_NSEC defined but unknown struct stat nanosecond type
      |   ^~~~~
make[1]: *** [/usr/lib/R/etc/Makeconf:169: libgit2/src/alloc.o] Error 1
make[1]: Leaving directory '/home/user/Downloads/git2r/src'
ERROR: compilation failed for package ‘git2r’
* removing ‘/usr/local/lib/R/site-library/git2r’
make: *** [Makefile:10: install] Error 1
```

**Comment by stewid** [comment]
Thanks for reporting this.

**Comment by stewid** [comment]
`git2r` can be built with a system installation of libgit2 instead of using the bundled source of libgit2. Therefore, it should work if you first install `libgit2` using:

- libgit2-dev   (package on e.g. Debian and Ubuntu)
- libgit2-devel (package on e.g. Fedora, CentOS and RHEL)
- libgit2       (Homebrew package on OS X)

Could you please try to install `git2r` after installing `libgit2`?

**Comment by blasif** [comment]
I had the same issue. Installing libgit2 worked for me.

**Comment by MichaelChirico** [comment]
Thanks for the suggested fix! It would be great if installation could fail more gracefully here :)

**Comment by stewid** [comment]
The build configuration script has been fixed and a new version (0.32.0) is now on CRAN
